src/ROUTES.ts

```diff
@@ -92,12 +92,12 @@ const ROUTES = {
     SETTINGS_APP_DOWNLOAD_LINKS: 'settings/about/app-download-links',
     SETTINGS_WALLET: 'settings/wallet',
     SETTINGS_WALLET_DOMAINCARD: {
-        route: 'settings/wallet/card/:domain',
-        getRoute: (domain: string) => `settings/wallet/card/${domain}` as const,
+        route: 'settings/wallet/card/:domain/:cardID?',
```

Comment:
There were some cases when we need cardID. Here's one:
We have 3 cards from the same domain: 1 physical named CARD1, 1 virtual and **assigned by admin** named CARD2, 1 virtual and **not assigned by admin** named CARD3. 
CARD1 and CARD3 are grouped under the domain in the same ExpensifyCardPage. To create a separate ExpensifyCardPage for CARD2 we need to pass cardID to differentiate from CARD1 and CARD3 ExpensifyCardPage.
There could possibly be a case when we have two admin issued cards from the same domain, then we also need cardID to make separate ExpensifyCardPage for each one.

Reply:
Oh! The need to display different card pages for the same domain is a valid use case. Thanks for pointing this out. 
In that case, we may always want to use `cardID` in the domain route. What do you think?

Otherwise, this would cause the following issues:

1) Inconsistency: In some cases, we would end up with some routes with `cardID` and some without. In the test video below, the route for the `cathycroissants.com` domain does not contain `cardID` whereas `cathycinamons.com` does.

https://github.com/Expensify/App/assets/3004659/74e6b6b6-3f10-44ec-96aa-4c92d9e61f1d

2) Deep-link URL to the route without `cardID` will fail

https://github.com/Expensify/App/assets/3004659/c8713302-be64-4078-8eac-4c554eb83cb7

Reply:
Yeah, I think that would work, I would only delete domain route and leave cardID route, since we won't use domain. What do you think?

Reply:
Hmm.. removing `domain` from `route` seems a good idea. 
I will review the latest changes and share the update later today.
And sorry. I missed reviewing this comment yesterday.